bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -353,6 +356,10 @@ impl RoomListItem {
         self.inner.inner_room().canonical_alias().map(|alias| alias.to_string())
     }
 
+    pub async fn room_info(&self) -> Result<RoomInfo, ClientError> {
+        Ok(RoomInfo::new(&self.inner).await?)
+    }
```

Comment:
What about removing `name`, `avatar_url`, `is_direct`, `canonical_alias` and other methods that expose data that are now accessible from `RoomInfo`?

Reply:
Let's talk about that in a while, when those are no longer used by both Android & iOS.

Reply:
I don't think UniFFI supports deprecating things such that the foreign code will get deprecation warnings.